### PR TITLE
Update governance-app.md

### PR DESCRIPTION
**Pull request by hanwencheng**
This PR will add the emergency proposal notification into the app design.

@rrtti  please feel free to check if the requirements meet your initial idea.

@geastwood please help fill the technical implementation and expectation time into the table. Thanks a lot!

### CODE REVIEW

File: Litentry/governance-app.md [hunk 1]
```diff
@@ -100,47 +106,49 @@ The technical implementation does not have much difference when we compare Polka
...
-|milestone 3 | 25	|70	|95	|1330|	6650|
-|milestone 4 | 20	|62	|82	|1148|	5740|
-|sum         | 105  |298|403|5642| 28210|
+| Milestone   | Research/Design (Hours) | Development (Hours) | Total (Hours) | DOT (Unit) | Dollar Value |
+|-------------|-------------------------|---------------------|---------------|------------|--------------|
+| milestone 1 | 32                      | 90                  | 122           | 1708       | 8540         |
+| milestone 2 | 28                      | 102                 | 139           | 1946       | 9730         |
+| milestone 3 | 25                      | 70                  | 95            | 1330       | 6650         |
+| milestone 4 | 20                      | 62                  | 82            | 1148       | 5740         |
+| sum         | 105                     | 298                 | 403           | 5642       | 28210        |
```
Comment (hanwencheng) [inline Litentry/governance-app.md]: The total number seems not updated.

**Review by rrtti — COMMENTED** [review]
is the alert app added as a new milestone for development or where have you added it?

File: Litentry/governance-app.md (hunk 1 above)
Comment (geastwood) [reply Litentry/governance-app.md]: Indeed, updated with new commit.

**Comment by geastwood** [comment]
> is the alert app added as a new milestone for development or where have you added it?

If I understand it correctly, by `Alert`, you mean remote *Push Notification*, similar to receiving an WhatsApp message and showing up on the Notification center.

This has been the one of the main objectives of this App, and what new is to offer fine-tuned ruleset (configuration), e.g. who will receive what under which conditions.

That's what **2.4** is for.

**Comment by rrtti** [comment]
@geastwood the app should not only send msgs, but also alert councillors they re needed by someone on the council sending a remark call t them that would activate an alarm on their phones: thats the original idea of a feature like the one we re thinking here.

**Review by rrtti — APPROVED** [review]

**Review by rrtti — APPROVED** [review]

**Pull request merged**